test/utils/testconfig/testconfig_test.go

```diff
@@ -260,27 +279,24 @@ func mockNode(name string) *corev1.Node {
 	aNode := corev1.Node{}
 	aNode.Name = name
 	aNode.Labels = make(map[string]string)
-	aNode.Labels["label1"] = ""
+	aNode.Labels[utils.PtpClockUnderTestNodeLabel] = ""
 	return &aNode
 }
 func GeneratePTPObjects(mode PTPMode) {
 	testclient.ClearTestClientsHolder()
 	switch mode {
 	case OrdinaryClock:
 		var mockClientObjects []runtime.Object
-		mockClientObjects = append(mockClientObjects, mockPtpConfig("config1", "namespace1", ptpv1.Master, OrdinaryClock))
-		mockClientObjects = append(mockClientObjects, mockPtpConfig("config2", "namespace1", ptpv1.Slave, OrdinaryClock))
+		mockClientObjects = append(mockClientObjects, mockPtpConfig("config1", "namespace1", ptpv1.Slave, OrdinaryClock))
```

Comment:
const , since config1 and namespace1 are repeated multiple times

Reply:
fixed